test/routes/licenceConditions.test.ts

```diff
@@ -250,25 +250,73 @@ describe('/hdc/licenceConditions', () => {
-      test(`sets conditions version if not previously set and additional conditions selected`, () => {
+    test(`passes postRelease true if agencyLocationId is out`, async () => {
+      const licenceService = createLicenceServiceStub()
+      const prisonerService = createPrisonerServiceStub()
+      prisonerService.getPrisonerPersonalDetails.mockResolvedValue({ agencyLocationId: 'out' })
+      const app = createApp({ licenceService, conditionsService, prisonerService }, 'roUser')
+
+      await request(app)
+        .post('/hdc/licenceConditions/standard/1')
+        .send({ additionalConditionsRequired: 'Yes', bookingId: 1 })
+        .expect(302)
+        .expect(() => {
+          expect(licenceService.update).toHaveBeenCalled()
+          expect(licenceService.update).toHaveBeenCalledWith({
+            bookingId: '1',
+            originalLicence: { licence: { key: 'value' } },
+            config: formConfig.standard,
+            userInput: { additionalConditionsRequired: 'Yes', bookingId: 1 },
+            licenceSection: 'licenceConditions',
+            formName: 'standard',
+            postRelease: true,
+          })
+        })
+    })
+  })
+
+  describe('POST /hdc/licenceConditions/additional/:bookingId', () => {
+    const routes = [
+      {
+        url: '/hdc/licenceConditions/additionalConditions/1',
+        body: { bespokeDecision: 'Yes', bookingId: 1, bespokeConditions: [{ text: 'a bespoke condition' }] },
+        nextPath: '/hdc/licenceConditions/conditionsSummary/1',
+      },
+      {
+        url: '/hdc/licenceConditions/additionalConditions/change/1',
+        body: { bespokeDecision: 'Yes', bookingId: 1, bespokeConditions: [{ text: 'a bespoke condition' }] },
+        nextPath: '/hdc/licenceConditions/conditionsSummary/change/1',
+      },
+    ]
+
+    routes.forEach((route) => {
+      test(`redirects to '${route.nextPath}' page for ${route.url}`, () => {
         const licenceService = createLicenceServiceStub()
         licenceService.getLicence.mockResolvedValue({
           licence: { key: 'value' },
-          versionDetails: { conditions_version: undefined },
+          versionDetails: { conditions_version: 1 },
         })
+        conditionsService.createConditionsObjectForLicence.mockReturnValue({})
         const app = createApp({ licenceService, conditionsService }, 'roUser')
 
         return request(app)
-          .post('/hdc/licenceConditions/standard/2')
-          .send({ additionalConditionsRequired: 'Yes', bookingId: 1 })
+          .post(route.url)
+          .send(route.body)
           .expect(302)
-          .expect(() => {
-            expect(licenceService.setConditionsVersion).toHaveBeenCalledWith(2, 1)
+          .expect((res) => {
+            expect(licenceService.updateLicenceConditions).toHaveBeenCalledWith(
+              1,
+              { licence: { key: 'value' }, versionDetails: { conditions_version: 1 } },
+              {},
```

Comment:
Should this be `additional_conditions_version`? See licenceService.ts LicenceRecord interface

Reply:
fixed these